lib/teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Since you are using the colon notation (:) for your `parent_permission`, to avoid errors you need to pass it properly in this `super` method, like so:

  ```
  super(age, name, parent_permission: parent_permission)
  ```